src/allmydata/scripts/create_node.py

```diff
@@ -344,6 +348,74 @@ def create_node(config):
         os.mkdir(basedir)
     write_tac(basedir, "client")
 
+    # if we're doing magic-wormhole stuff, do it now
+    if config['join'] is not None:
+        print >>out, "Opening wormhole with code '{}'".format(config['join'])
+        relay_url = config.parent['wormhole-server']
+        print >>out, "Connecting to '{}'".format(relay_url)
+
+        wh = wormhole.create(
+            appid=u"tahoe-lafs.org/lafs",
```

Comment:
(But yes the main point about the appid being wrong/not-matching above needs to be fixed)